packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -405,10 +405,8 @@ message Invitation {
   enum Type {
     /// Requires both to be online to complete key exchange.
     INTERACTIVE = 0;
-    /// Guest's identity key is known; invitation can be accepted by any valid peer.
+    /// Invitation can be accepted by any valid peer.
     OFFLINE = 1;
-    /// Multiple-use interactive invitation.
-    MULTIUSE = 2;
```

Comment:
if removing, perhaps this should be deprecated first so that we retain compatibility for a few versions?

Reply:
though...would need to consider whether the implementation supported that as well. I'm just wary of this breaking invitations between versions of shell

Reply:
after creation we seem to rely on authCode presence, not type

Reply:
but doesn't this all get encoded into the invitation code? would old implementations decode it properly?

Reply:
right. I think protobuf maps uknown enum to 0 so we'd just use multiUse flag. 
but yeah, I'll bring it back with a deprecation comment